tests/unit_tests/cpu/tools_navigator.cpp

```diff
@@ -212,19 +212,19 @@ GTEST_TEST(detray_propagator, navigator) {
     std::map<dindex, std::vector<dindex>> sf_sequences;
 
     // layer 1
-    sf_sequences[7] = {594u, 491u, 475u, 492u, 476u, 595u};
+    sf_sequences[7] = {370u, 495u, 479u, 496u, 480u, 371u};
```

Comment:
Why does indices change here?

Reply:
The same reason they changed in the toy detector test: I put the portals in front of the sensitives into the surface lookup